qube/quantitative/ta/indicators.py

```diff
@@ -1447,3 +1448,27 @@ def psar(ohlc, iaf=0.02, maxaf=0.2):
     psar_i, psarbear, psarbull = __psar(ohlc['close'].values, ohlc['high'].values, ohlc['low'].values, iaf, maxaf)
 
     return pd.DataFrame({"psar": psar_i, "up": psarbear, "down": psarbull}, index=ohlc.index)
+
+def fdi(x, e_period = 30):
```

Comment:
It'd be better to add types annotations here, x : Union[Series, .....] etc.

Reply:
Done

Reply:
Done